descope/sdk/mgmt.go

```diff
@@ -241,6 +241,14 @@ type User interface {
 	// Returns slice of users and total number of users for the query
 	SearchAll(ctx context.Context, options *descope.UserSearchOptions) ([]*descope.UserResponse, int, error)
 
+	// Search all test users according to given filters
+	//
+	// The options optional parameter allows to fine-tune the search filters
+	// and results. Using nil will result in a filter-less query with a set amount of
+	// results.
+	// Returns slice of users and total number of users for the query
+	SearchAllTest(ctx context.Context, options *descope.UserSearchOptions) ([]*descope.UserResponse, int, error)
```

Comment:
I would change name to SearchAllTestUsers

Reply:
Also, .need to add to the readme in the test users section

![image](https://github.com/user-attachments/assets/f9c37fd9-b120-4a62-9eb1-b1da1029a358)

Reply:
Changed.